src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/api/BundleTests.kt

```diff
@@ -33,7 +33,7 @@ class BundlePackTests : IntegrationTest()
     {
         val response = apiRequestHelper.post("/bundle/pack/minimal/", emptyMap(), ContentTypes.zip, fakeGlobalReportReviewer())
         assertThat(response.statusCode).isEqualTo(200)
-        assertThat(response.headers["content-type"]).isEqualTo("application/zip")
+        assertThat(response.headers["content-type"]).isEqualTo("application/octet-stream")
```

Comment:
Not sure why this change has happened?

Reply:
Now simply passing through the response from `orderly.server` (including all headers) and this is what `orderly.server` returns for this endpoint.